robot/EDA/resources/CourseConnectionsSettingsPageObject.py

```diff
@@ -48,13 +50,13 @@ def verify_dropdown_values(self, field, *args):
         """ Verify the field has the drop down values given by *args
             we have to pass all the values available in the UI since the count is also verified
         """
-        locator = eda_lex_locators["eda_settings_cc"]["dropdown_values"]
         locator_count = eda_lex_locators["eda_settings_cc"]["dropdown_values_count"].format(field)
+        self.selenium.wait_until_page_contains_element(locator_count, error=f"Dropdown field is not available in edit mode")
```

Comment:
You don't need to add an "f" here since you aren't formatting the error message here. (Just FYI, you can leave it as-is for now)